org.eclipse.gef.examples.ui.pde/src/org/eclipse/gef/examples/ui/pde/internal/wizards/ProjectUnzipperNewWizard.java

```diff
@@ -182,61 +176,36 @@ public boolean performFinish() {
 				@Override
 				public void execute(IProgressMonitor monitor) throws InterruptedException {
 					try {
-						monitor.beginTask(Messages.monitor_creatingProject, 120);
+						SubMonitor subMonitor = SubMonitor.convert(monitor, Messages.monitor_creatingProject, 150);
```

Comment:
I just noticed that this was also wrong in the original implementation... The ticks are consumed within a for-loop, so 150 (140 without line 207) are only enough if there is a single format.
```suggestion
						SubMonitor subMonitor = SubMonitor.convert(monitor, Messages.monitor_creatingProject, 140 * nameFormats.length);
```